testutils.py

```diff
@@ -51,3 +51,16 @@ def if_rollback():
     if "ROLLBACK TARGET" in out:
         return True
     return False
+
+def get_fedora_release():
+    "Finds the fedora version in str"
+    ver = ""
+    with open("/etc/os-release") as fobj:
```

Comment:
os-release is supposed to be for machine reading. That is why I chose to use this.